pkg/apis/extensions/types.go

```diff
@@ -349,31 +340,35 @@ type DaemonSetSpec struct {
 	// More info: http://kubernetes.io/docs/user-guide/replication-controller#pod-template
 	Template api.PodTemplateSpec `json:"template"`
 
-	// TODO(madhusudancs): Uncomment while implementing DaemonSet updates.
-	/* Commenting out for v1.2. We are planning to bring these fields back with a more robust DaemonSet update implementation in v1.3, hence not deleting but just commenting these fields out.
+	// Indicates that the DaemonSet is paused and will not be processed by the
+	// DaemonSet controller.
+	Paused bool `json:"paused,omitempty"`
+
+	// Indicates that the DaemonSet will automaticly update pods when DaemonSet Spec was changed.
+	AutoUpdate bool `json:"autoUpdate,omitempty"`
```

Comment:
How about instead of a separate field, we default Paused to true? Paused for Deployments means exactly the opposite of AutoUpdate.